lua/entities/gmod_tardis_part/shared.lua

```diff
@@ -37,4 +37,18 @@ end)
 
 hook.Add("CanDrive", "tardis-part", function(ply,ent)
     if ent.TardisPart then return false end
-end)
\ No newline at end of file
+end)
+
+function ENT:SetVisible(visible)
```

Comment:
Why has this been moved to shared? It can only effectively work serverside (unless you're spamming it every frame clientside which is bad for performance)

Reply:
It created some confusion in extensions
The second most common use-case for this is in the custom hooks, having to specify `if SERVER then` just for that each time is not convenient

Reply:
I disagree with this, I think having it in shared code could cause even more problems and confusion in that SetColor does not persist clientside unless you're spamming it every frame which is also a big performance hit. I'm not sure under what scenarios you'd generally need to specify `if SERVER then` anyway here, as the Use code will generally be called serverside only, and as I mentioned we should not be calling this in a clientside Think hook.